src/Host.cpp

```diff
@@ -369,6 +369,36 @@ void Host::resetProfile()
     qDebug()<<"resetProfile() DONE";
 }
 
+// Saves profile to disk - does not save items dirty in the editor, however.
+// takes a directory to save in or an empty string for the default location
+// as well as a boolean whenever to sync the modules or not
+// returns true+filepath if successful or false+error message otherwise
+std::tuple<bool, QString, QString> Host::saveProfile(const QString& saveLocation, bool syncModules)
+{
+    QString directory_xml;
+    if (saveLocation.isEmpty()) {
+        directory_xml = QDir::homePath() + QLatin1String("/.config/mudlet/profiles/") + getName() + QLatin1String("/current");
+    } else {
+        directory_xml = saveLocation;
+    }
+
+    QString filename_xml = directory_xml + "/" + QDateTime::currentDateTime().toString("dd-MM-yyyy#hh-mm-ss") + ".xml";
```

Comment:
... only in the _display_ of profile saved with that new format of date-time - they will still be listed in the correct order and the most recent one loaded by default but will appear as empty/blank entries in the drop-down list on those old versions... :neutral_face: